assets/src/edit-story/panels/shared.js

```diff
@@ -9,6 +9,13 @@ import PropTypes from 'prop-types';
  */
 import { __ } from '@wordpress/i18n';
 
+export const RowPanel = styled.form`
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
```

Comment:
When allowing the elements to take up full space as instructed above, use a spacing span in between the elements to account for the padding - just a 12px span should do it. Then this setting isn't required (or usefull) anymore.

Reply:
Did you get the spacing to work correctly?